ProjectWithUMLDiagram/src/com/beyond/person/core/enums/DancerType.java

```diff
@@ -0,0 +1,7 @@
+package com.beyond.person.core.enums;
```

Comment:
@AraChilingaryan package name should be singular, as enum reserved word you can use enumaration

Reply:
done